### PR TITLE
docs: updated readme and manifest

### PR DESCRIPTION
**Pull request by davidbp**

### CODE REVIEW

File: encoders/image/CustomImageTorchEncoder/manifest.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ manifest_version: 1
...
 kind: encoder
 type: pod
-description: '`CustomImageTorchEncoder` encodes data from a ndarray, potentially B
-  x (Channel x Height x Width) into a ndarray of `B x D`.'
+description: '`CustomImageTorchEncoder` encodes data from a Numpy `ndarray` of shape  `BatchSize
+  x (Channel x Height x Width)` into  `BatchSize x EmbeddingDimension` Numpy array.'
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
```
Comment (JoanFM) [inline encoders/image/CustomImageTorchEncoder/manifest.yml:10]: make this documentation link point to the README.md

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: encoders/image/CustomImageTorchEncoder/manifest.yml (hunk 1 above)
Comment (davidbp) [reply encoders/image/CustomImageTorchEncoder/manifest.yml:10]: do you mean a link to `encoders/image/CustomImageTorchEncoder/README.md`?

Comment (JoanFM) [reply encoders/image/CustomImageTorchEncoder/manifest.yml:10]: yes, but with the url of the github itself.

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: encoders/image/CustomImageTorchEncoder/README.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+jina pod --uses custom_folder/customimagetorchencoder.yml --port-in 55555 --port-out 55556
+```
+
+5. Docker command, Specify the image name along with the version tag. The snippet below uses Jina version 1.0.4
+
+```
+ docker pull jinahub/pod.encoder.customimagetorchencoder:0.0.11-1.0.4
+```
+
+##
```
Comment (JoanFM) [inline encoders/image/CustomImageTorchEncoder/README.md:50]: is this last line needed?

**Review by JoanFM — APPROVED** [review]

**Pull request merged**